const createErroror = require("../utils/createerroror");
exports.createExperience = async () => {
  try {
  } catch (error) {
    next(error);
  }
};

exports.updateExperience = async () => {
  try {
  } catch (error) {
    next(error);
  }
};
exports.deleteExperience = async () => {
  try {
  } catch (error) {
    next(error);
  }
};
